CONTRIBUTING.md

```diff
@@ -1,19 +1,47 @@
-# Don't use the issue tracker to ask questions
```

Comment:
As someone not doing their first fork, all this boilerplate setup distracts from the actual 'useful' content like how acceptance tests work / how to run with different rails versions. Can it move into 1 heading or maybe just link to a 'Contributing to a ruby open source project' section / wiki page ?